hooks/useFileUpload.tsx

```diff
@@ -26,11 +26,11 @@ const useFileUpload = () => {
         throw new Error(`File size exceeds the maximum limit of ${MAX_FILE_SIZE / ONE_MB}MB.`)
       }
       const { uri } = await uploadFile(file)
-      setMimeType(mimeType)
```

Comment:
if `animationUri` is not provided, `mimeType` should be set to image
- actual: mimeType will only be saved to `metadata` in IPFS if `animationUri` is provided. if only image, mimeType will be missing from metadata.
- expected: mimeType is present in 100% of metadata. including case where only image is selected.